pyproject.toml

```
@@ -31,3 +31,7 @@ lxml = "4.6.3"
 [tool.poetry.scripts]
 kalamine = "kalamine.cli:make"
 xkalamine = "kalamine.cli_xkb:cli"
+
+[build-system]
+requires = ["poetry>=0.12"]
+build-backend = "poetry.masonry.api"
```

Source: https://python-poetry.org/docs/pyproject/#poetry-and-pep-517

```suggestion
requires = ["poetry-core>=1.0.0"]
build-backend = "poetry.core.masonry.api"
```